### PR TITLE
[EASI-3713] - Frequency Updates Part 4

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# EASI-3713

## Changes and Description

This PR is part 4 of all the frequency updates to forms. 

- Updated backend schema to use new `FrequencyTypeNew` for `dataSharingFrequency` and `dataCollectionFrequency`
- Added implementation of `FrequencyForm` to `opsEvalAndLearning.dataSharingFrequency` and `opsEvalAndLearning.dataCollectionFrequency`  form
- Added `dataSharingFrequency` and `dataCollectionFrequency` to readonly
- Updated tests
- Updated cypress test

## How to test this change

- Navigate to second to last page for opsEvalAndLearning (where `dataSharingFrequency` question is rendered)
- Verify correct render and form functionality according to Figma
- Verify the data persists and renders correctly in the readonly view
- Verify the data renders in the CSV

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/graph/schema.graphql [hunk 1]
```diff
@@ -2036,17 +2040,18 @@ enum ConfidenceType {
...
+#   ANNUALLY
+#   BIANNUALLY
+#   QUARTERLY
+#   MONTHLY
+#   SEMI_MONTHLY
+#   WEEKLY
+#   DAILY
+#   OTHER
+#   NOT_PLANNING_TO_DO_THIS
+# }
```
Comment (garyjzhao) [inline pkg/graph/schema.graphql:2054]: leftover commented code

**Review by garyjzhao — CHANGES_REQUESTED** [review]
Looks good. Just one thing

File: pkg/graph/schema.graphql (hunk 1 above)
Comment (patrickseguraoddball) [reply pkg/graph/schema.graphql:2054]: We are leaving that in intentionally for Toms work

**Review by garyjzhao — APPROVED** [review]
LGTM

**Pull request merged**